spanner/opentelemetry_traces/src/main/java/com/example/spanner/OpenTelemetryUsage.java

```diff
@@ -0,0 +1,152 @@
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+
+/**
+ * This sample demonstrates how to configure OpenTelemetry and inject via Spanner Options.
+ */
+public class OpenTelemetryUsage {
+
+  // TODO(developer): Replace these variables before running the sample.
+  static String projectId = "my-project";
+  static String instanceId = "my-instance";
+  static String databaseId = "my-database";
+  public static SdkTracerProvider sdkTracerProvider;
+
+  public static void main(String[] args) {
+
+
+    // Use getSpannerWithCloudTraceExporter instead of getSpannerWithOTLPExporter to directly
+    // export to Observability backend using Cloud Trace exporter.
+    // Spanner spanner = getSpannerWithCloudTraceExporter();
+    Spanner spanner = getSpannerWithOtlpExporter();
+
+    DatabaseClient dbClient = spanner
+        .getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+
+    try (ResultSet resultSet =
+        dbClient
+            .singleUse() // Execute a single read or query against Cloud Spanner.
+            .executeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"))) {
+      while (resultSet.next()) {
+        System.out.printf(
+            "%d %d %s", resultSet.getLong(0), resultSet.getLong(1), resultSet.getString(2));
+      }
+    }
+
+    sdkTracerProvider.forceFlush();
+  }
+
+  public static Spanner getSpannerWithOtlpExporter() {
+    // [START spanner_opentelemetry_traces_otlp_usage]
+    Resource resource = Resource
+        .getDefault().merge(Resource.builder().put("service.name", "My App").build());
+
+    OtlpGrpcSpanExporter otlpGrpcSpanExporter =
+        OtlpGrpcSpanExporter
+            .builder()
+            .setEndpoint("http://localhost:4317") // Replace with your OTLP endpoint
+            .build();
+
+    // Using a batch span processor
+    // You can use `.setScheduleDelay()`, `.setExporterTimeout()`,
+    // `.setMaxQueueSize`(), and `.setMaxExportBatchSize()` to further customize.
+    BatchSpanProcessor otlpGrpcSpanProcessor =
+        BatchSpanProcessor.builder(otlpGrpcSpanExporter).build();
+
+    // Create a new tracer provider
+    sdkTracerProvider = SdkTracerProvider.builder()
+        // Use Otlp exporter or any other exporter of your choice.
+        .addSpanProcessor(otlpGrpcSpanProcessor)
+        .setResource(resource)
+        .setSampler(Sampler.traceIdRatioBased(0.1))
+        .build();
+
+    // Export to a collector that is expecting OTLP using gRPC.
+    OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+        .setTracerProvider(sdkTracerProvider).build();
+
+    // Enable OpenTelemetry traces before Injecting OpenTelemetry
+    SpannerOptions.enableOpenTelemetryTraces();
+
+    // Inject OpenTelemetry object via Spanner options or register as GlobalOpenTelemetry.
```

Comment:
nit: Let's mention "OpenTelemetrySdk.builder()....buildAndRegisterGlobal();" in the comment to show how global registration could be done.